### ISSUE TITLE
azure-quickstart-templates/101-logic-app-ftp-to-blob 

### ISSUE THREAD

**Issue by marc-bouvier** [Azure/azure-quickstart-templates#8277]
Hi, 
While trying to automate Logic App resrouce creation, I had a problem with the parameters to provide to the SFTP connector.

https://github.com/Azure/azure-quickstart-templates/blob/1d5a41e1ec6125d147b829f4c830c312f3b7e971/101-logic-app-ftp-to-blob/azuredeploy.json#L98-L106

When using [SFTP - SSH connector](https://docs.microsoft.com/fr-fr/azure/connectors/connectors-sftp-ssh), the parameters  names to be provided are : 

- hostName
- userName
- password
- portNumber
- acceptAnySshHostKey
- sshHostKeyFingerprint
- rootFolder

I didn't find anywhere in the documentation where those parameters are documented. I had to watch http xhr requests in my browser to find the name of the parameters to provide for my connection resource.

Since the old [SFTP connector](https://docs.microsoft.com/fr-fr/azure/connectors/connectors-create-api-sftp) is deprecated, I believe the template might need to be updated (maybe with comments or alternative template for legacy connector).

Cheers.

**Comment by bmoore-msft** [comment]
@jeffhollan any ideas

**Comment by jeffhollan** [comment]
Yeah looks like the connector had a major version change from when I created this. Looks like a few of the properties got renamed and some added. I would assign to Jon Fancey - I don’t see his handle pop up on this repo to tag

**Comment by bmoore-msft** [comment]
@jonfancey - is this something you can help with?
